File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program95.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept character from user and display its ASCII value in decimal, octal and hexadecimal format.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{

	public char ch;

	public Solution(char ch)
	{
		this.ch = ch;
	}

	public void Display()
	{	
		
		int iCnt=ch;
		System.out.println("Decimal		:"+iCnt);
		System.out.println("Octal		:"+(Integer.toOctalString(iCnt)));
		System.out.println("HexaDecimal		:"+(Integer.toHexString(iCnt)));
		
	}


}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);

		System.out.println("Enter character:");
		char ch = sObj.next().charAt(0);

		Solution obj = new Solution(ch);
		obj.Display();
	}
}

/*
OUTPUT:
Enter character:
A
Decimal		:65
Octal		:101
HexaDecimal		:41


*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program95.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept character from user and display its ASCII value in decimal, octal and hexadecimal format.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class Solution
{
	private:
		char ch;
		
	public:
		Solution(char ch='\0')
		{
			this->ch = ch;
		}

		void Display()
		{
			int iAns = ch;
			cout<<"Decimal		:"<<iAns<<"\n";
			cout<<"Octal		:"<<oct<<iAns<<"\n";
			cout<<"HExadecimal	:"<<hex<<iAns<<"\n";
			
		}


};

int main()
{
	char ch='\0';

	cout<<"Enter the character: ";
	cin>>ch;

	Solution *dObj = new Solution(ch);
	dObj->Display();
	
	delete dObj;
	return 0;
}

/*
OUTPUT:

Enter the character: A
Decimal		:65
Octal		:101
HExadecimal	:41

*/
